components/generic/ContentCard.stories.js

```diff
@@ -16,86 +16,14 @@ storiesOf('Generic', module)
       />
     </b-col>`
   }))
-  .add('Section with content cards', () => ({
-    components: { ContentCardSection },
-    i18n: new VueI18n(),
-    data() {
-      return {
-        section: {
-          fields: {
-            headline: 'Art',
-            text: 'Discover inspiring art, artists and stories in 2,165,362 artworks from European museums, galleries, libraries and archives',
-            hasPart: [{
-              sys: {
-                id: '123456'
-              },
-              fields: {
-                name: 'Jonas Martinaitis. Piemenų landynė. 1942 | Jonas Martinaitis',
-                image: {
-                  fields: {
-                    file: {
-                      url: 'img/landscape.jpg'
-                    }
-                  }
-                },
-                url: 'http://europeana.eu'
-              }
-            },
-            {
-              sys: {
-                id: '6789'
-              },
-              fields: {
-                name: 'Kanutas Ruseckas. Mitologinė pastoralinė scena. XIX a. | Kanutas Ruseckas',
-                image: {
-                  fields: {
-                    file: {
-                      url: 'img/landscape.jpg'
-                    }
-                  }
-                },
-                url: 'http://europeana.eu'
-              }
-            },
-            {
-              sys: {
-                id: '98765'
-              },
-              fields: {
-                name: 'Jonas Mackevičius. Anykščių bažnyčia. XX a. I p. | Jonas Mackevičius',
-                image: {
-                  fields: {
-                    file: {
-                      url: 'img/landscape.jpg'
-                    }
-                  }
-                },
-                url: 'http://europeana.eu'
-              }
-            },
-            {
-              sys: {
-                id: '43210'
-              },
-              fields: {
-                name: 'Juozapas Kamarauskas. Vilniaus muitinės pastatas. 1920 | Juozapas Kamarauskas',
-                image: {
-                  fields: {
-                    file: {
-                      url: 'img/landscape.jpg'
-                    }
-                  }
-                },
-                url: 'http://europeana.eu'
-              }
-            }]
-          }
-        }
-      };
-    },
-    template: `<b-container class="mt-3">
-      <ContentCardSection 
-        :section="section"
+  .add('Mini card', () => ({
```

Comment:
Could you add all variants? 
These are: default, entity, mini, list. See also: https://github.com/europeana/portal.js/blob/197d17d020bbd56c22de40d24ac20ab3de91a7f3/components/generic/ContentCard.vue#L137